### DISCUSSION TITLE
Issue loading from python script

### DISCUSSION THREAD

**Discussion by efleurant** [ggarra13/mrv2#245] [Q&A]
Hi,

I'm trying to load a exr sequence with a python script using the `-pythonScript` argument, but it ends up crashing (segfault ). If I run the snippet from mrv2's python console though, it works as expected. Here's the code snippet:
```python
import mrv2
from mrv2 import cmd, image

mrv2.image.setOcioIcs("ACES - ACEScg")
mrv2.image.setOcioView("Rec.709 (ACES)")
mrv2.cmd.open(filename="/path/to/my/sequence/my_shot_name.000100.exr")
```

Note that `/path/to/my/sequence/my_shot_name.000100.exr` being the first frame of my sequence... in the UI that seems to trigger the whole sequence. I tried using the standard sequence pattern ("%06d", "######") but those didn't work.

Here's the error I'm getting when running the above code with through the `-pythonsScript`:
```
GOT SIGNAL 11
0x7fa20503245c cfunction_call
 (Objects/methodobject.c:542)
0x7fa20500b82e _PyObject_MakeTpCall
 (Objects/call.c:214)
0x7fa20506a97c _PyEval_EvalFrameDefault
 (Python/ceval.c:4769)
0x7fa2050695da _PyEval_EvalFrame
 (./Include/internal/pycore_ceval.h:73)
0x7fa2050695da _PyEval_Vector
 (Python/ceval.c:6434)
0x7fa2050e5979 PyEval_EvalCode
 (Python/ceval.c:1148)
0x7fa2050f9fc2 run_eval_code_obj
 (Python/pythonrun.c:1741)
0x7fa2050f9f49 run_mod
 (Python/pythonrun.c:1762)
0x7fa2050f9eb8 PyRun_StringFlags
```

Am I missing something? thanks in advance...

**Comment by ggarra13** [comment]
Hi, @efleurant!  -pythonScript currently runs without a UI so any loading of an image that tries to play it back will currently lead to a crash.
What's your use case scenario?  Do you want to bake-in OCIO transforms?

**Comment by ggarra13** [comment]
@efleurant I added the functionality to support most python commands using -pythonScript.  Please download v1.1.9 beta and give it a try from: 

https://sourceforge.net/projects/mrv2/files/beta/

I tested with the built-in OCIO studio config and this script:

```
#!/usr/bin/env python

import mrv2
from mrv2 import cmd, image

image.setOcioIcs("ACEScg")
image.setOcioView("ACES 1.0 - SDR Video")
cmd.open("/home/gga/Pictures/Mantaflow_v09/Fluid.0001.exr")
cmd.save("/home/gga/Test.mov")
```

I noticed save uses "file" instead of "filename" so I changed that in the api.
